opencog/query/PatternUtils.cc

```diff
@@ -117,21 +116,18 @@ void get_connected_components(
 			size_t nc = components.size();
 			for (size_t i = 0; i<nc; i++)
 			{
-				std::set<Handle> cur_vars(component_vars[i]);
```

Comment:
Oh, well I start to understand why constness isn't supported so well, turns out constness isn't very well supported in the atomspace code as well... It starts to look like a much bigger task...